jenkins_pipelines/scripts/kafka/README.md

```diff
@@ -0,0 +1,62 @@
+# Kafka automation concept
+
+Messaging system to speed up business processes via services API.
```

Comment:
what does that mean ?

Reply:
I used to much marketing here, corrected sentence is:
`Messaging system to automate manual tasks with SUSE internal services via API.`